code/app/build.gradle

```diff
@@ -63,13 +63,10 @@ dependencies {
 
     implementation project(':edgeidentity')
 
-    // TODO: Update these dependencies when all 2.0 artifacts are published to maven
-    // implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
-    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
-    implementation ('com.adobe.marketing.mobile:identity:2.0.0-SNAPSHOT') {
-        transitive = false
-    }
+    implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
+    implementation 'com.adobe.marketing.mobile:identity:2.0.0'
```

Comment:
`mavenDirectIdentityVersion=2.0.0`

implementation 'com.adobe.marketing.mobile:identity:${rootProject. mavenDirectIdentityVersion}'

Reply:
gradle.properties holds prod dependencies to declare min compatible version for current release. For tests/test app we often update to latest version available for quick tests, so I would prefer not lock that in a variable. what do you think?

Reply:
That makes sense. I wanted to see if we would like to test the app and integration tests with the same version. Should we fix to full specific version for tests instead of dynamic one?

Reply:
updated all test app dependencies to 2.+ so they work with the snapshot and we don't need to update later for 2.0